onnx/helper.py

```diff
@@ -1533,3 +1533,34 @@ def get_all_tensor_dtypes() -> KeysView[int]:
     :return: all tensor types from TensorProto
     """
     return mapping.TENSOR_TYPE_MAP.keys()
+
+
+attribute_proto_type_to_str = {
```

Comment:
So there's no way for us to get it from the protobuf emum object?

Reply:
Not that I can tell, but I'm relatively inexperienced with it comes to protobufs in Python.

Reply:
Turns out there is a way, did some exploring and found a cleaner way to do this.